src/doctest/build.zig

```diff
@@ -81,11 +81,14 @@ pub fn runBuild(
         },
     }
 
+    const target_query = try std.Target.Query.parse(.{
+        .arch_os_abi = cmd.target_str orelse "native",
+    });
+    const target = try std.zig.system.resolveTargetQuery(target_query);
+
     // Add link options
     for (cmd.link_objects) |link_object| {
-        // TODO: we're setting the obj file extension before parsing
-        //       the provided crosstarget string. Prob not ok.
```

Comment:
Based on this comment (and the meaning of the command being constructed), I assume what I did is the intended fix? (using the dynamic library suffix from the provided target, not the host target)